app/commands.py

```diff
@@ -195,48 +189,49 @@ def setup_commands(application):
-
-    """
-    Rebuild the data in ft_billing for the given service_id and date
-    """
-
-    def rebuild_ft_data(process_day, service):
-        deleted_rows = delete_billing_data_for_service_for_day(process_day, service)
-        current_app.logger.info(
-            f"deleted {deleted_rows} existing billing rows for {service} on {process_day}"
-        )
-        transit_data = fetch_billing_data_for_day(
-            process_day=process_day, service_id=service
-        )
-        # transit_data = every row that should exist
-        for data in transit_data:
-            # upsert existing rows
-            update_fact_billing(data, process_day)
-        current_app.logger.info(
-            f"added/updated {len(transit_data)} billing rows for {service} on {process_day}"
-        )
-
-    if service_id:
-        # confirm the service exists
-        dao_fetch_service_by_id(service_id)
-        rebuild_ft_data(day, service_id)
-    else:
-        services = get_service_ids_that_need_billing_populated(
-            get_midnight_in_utc(day), get_midnight_in_utc(day + timedelta(days=1))
-        )
-        for row in services:
-            rebuild_ft_data(day, row.service_id)
+# TODO maintainability what is the purpose of this command?  Who would use it and why?
+# COMMENTING OUT UNTIL WE DETERMINE IF WE NEED IT OR NOT
+# @notify_command(name="rebuild-ft-billing-for-day")
+# @click.option("-s", "--service_id", required=False, type=click.UUID)
+# @click.option(
+#     "-d",
+#     "--day",
+#     help="The date to recalculate, as YYYY-MM-DD",
+#     required=True,
+#     type=click_dt(format="%Y-%m-%d"),
+# )
+# def rebuild_ft_billing_for_day(service_id, day):
+
+#     """
+#     Rebuild the data in ft_billing for the given service_id and date
+#     """
+
+#     def rebuild_ft_data(process_day, service):
+#         deleted_rows = delete_billing_data_for_service_for_day(process_day, service)
+#         current_app.logger.info(
+#             f"deleted {deleted_rows} existing billing rows for {service} on {process_day}"
+#         )
+#         transit_data = fetch_billing_data_for_day(
+#             process_day=process_day, service_id=service
+#         )
+#         # transit_data = every row that should exist
+#         for data in transit_data:
+#             # upsert existing rows
+#             update_fact_billing(data, process_day)
+#         current_app.logger.info(
+#             f"added/updated {len(transit_data)} billing rows for {service} on {process_day}"
+#         )
+
+#     if service_id:
+#         # confirm the service exists
+#         dao_fetch_service_by_id(service_id)
+#         rebuild_ft_data(day, service_id)
+#     else:
+#         services = get_service_ids_that_need_billing_populated(
+#             get_midnight_in_utc(day), get_midnight_in_utc(day + timedelta(days=1))
+#         )
+#         for row in services:
+#             rebuild_ft_data(day, row.service_id)
```

Comment:
Nice - we probably could get rid of all of this though!  Our billing is going to be _vastly_ different from this, so let's not even try to repurpose it.